package.json

```diff
@@ -15,6 +15,7 @@
         "@reduxjs/toolkit": "^1.2.3",
         "@svgdotjs/svg.js": "^3.0.12",
         "@svgdotjs/svg.panzoom.js": "^2.1.2",
+        "@testing-library/react": "13.4.0",
```

Comment:
Not sure... we already have jest throught CRA, why adding another testing framework ?

Reply:
And there is no pakage-lock.json diff pushed then it won't work anyway.